scripts/qa_crowbarsetup.sh

```diff
@@ -1157,10 +1157,10 @@ function onadmin_bootstrapcrowbar
         else
             if iscloudver 7M6minus || [[ $cloudsource = mitakacloud7 ]] ; then
                 safely crowbar_api_request POST $crowbar_init_api /database/new \
-                    '--data username=crowbar&password=crowbar' "$crowbar_api_v2_header"
+                    '--data username=crowbar-nondefault&password=crowbar' "$crowbar_api_v2_header"
```

Comment:
Could you elaborate in the commit message what the problem is? Is it that the upgrade process overwrites the default username/password credentials? I don't quite understand... :slightly_frowning_face:

Reply:
@nicolasbock read the bug report I mentioned in the commit msg :)

Reply:
I did :smile: Ok, I'll read it again, maybe I'll get it then...

Reply:
I think, we should prefer to have a short summary of the bug in here, because bugs can be non-public or get lost later and we still want to understand why the change was done.